src/Illuminate/Database/Schema/Builder.php

```diff
@@ -94,6 +94,28 @@ public static function morphUsingUuids()
         return static::defaultMorphKeyType('uuid');
     }
 
+    /**
+     * Create a database in the schema if the database not exists.
+     *
+     * @param  string  $name
+     * @return bool
+     */
+    public function createDatabaseIfNotExists($name)
+    {
+        throw new LogicException('This database driver does not support create databases.');
+    }
+
+    /**
+     * Drop a database from the schema if the database exists.
+     *
+     * @param  string  $name
+     * @return bool
+     */
+    public function dropDatabaseIfExists($name)
+    {
+        throw new LogicException('This database driver does not support drop databases.');
```

Comment:
The message should be changed to `This database driver does not support dropping databases.`

Reply:
Could maybe be made its own exception class so the message is in one place, rather than throwing the same `LogicException` in multiple spots.

Reply:
@level39 Done. Thank you.

@martinbean We usually use "LogicException" for this kind of things.